### PR TITLE
[AGW documentation] Test tier creation for AGW Upgrades

### PR DESCRIPTION
**Pull request by jaredmullane**
Signed-off-by: Jared Mullane <jmullane@fb.com>

## Summary

Partners have been experiencing issues by accidentally upgrading all of their AGWs at once. I have updated the documentation to show how to set up a test tier where they can upgrade on AGW prior to rolling out updates at scale. 

## Test Plan

Ran docs/docusaurus/create_docusaurus_website.sh

### CODE REVIEW

File: docs/docusaurus/versioned_docs/version-1.1.0/lte/upgrade.md [hunk 1]
```diff
@@ -25,9 +25,11 @@ software version in this tab has changed. If it hasn't changed, something
...
 the AGW to troubleshoot the installation (see the next section).
 
-If you want to roll out an upgrade slowly to your fleet, you can segment your
-AGWs into different tiers and upgrade tiers one-by-one. Use the NMS to create
-new tiers and assign your AGWs to them.
+It is recommended that you conduct upgrades on a test prior to rolling it out to your entire network. 
+
+Setting up a test tier is a two part process. First, create a test tier in the "Equipment" tab by clicking the "Upgrade" button. Next, click the "Add" button in the top right corner and enter in your test tier information and click the "Save" button. 
+
+Once this is created, navigate back to the main "Equipment" tab page. Under the Gateways table select the "Actions" button on the row of the AGW you would like to add to the test tier.  Select the "Edit" option to take you into the configuration page and then select "Edit JSON" in the top right. At the bottom on the JSON object change the "tier" to match the name of your test tier. Click "Save" in the top right to complete the process.
```
Comment (karthiksubraveti) [inline docs/docusaurus/versioned_docs/version-1.1.0/lte/upgrade.md:32]: Selection of a tier for a equipment can be done from the Equipment overview page.

File: docs/docusaurus/versioned_docs/version-1.1.0/lte/upgrade.md [hunk 2]
```diff
@@ -25,9 +25,11 @@ software version in this tab has changed. If it hasn't changed, something
 probably went wrong with the autoupgrade. You will probably have to SSH into
 the AGW to troubleshoot the installation (see the next section).
 
-If you want to roll out an upgrade slowly to your fleet, you can segment your
-AGWs into different tiers and upgrade tiers one-by-one. Use the NMS to create
-new tiers and assign your AGWs to them.
+It is recommended that you conduct upgrades on a test prior to rolling it out to your entire network. 
+
```
Comment (karthiksubraveti) [inline docs/docusaurus/versioned_docs/version-1.1.0/lte/upgrade.md:29]: This is good documentation. However it will be really useful to do it with screenshot or a gif. Text description is very hard to follow.

File: docs/docusaurus/versioned_docs/version-1.1.0/lte/upgrade.md (hunk 1 above)
Comment (karthiksubraveti) [reply docs/docusaurus/versioned_docs/version-1.1.0/lte/upgrade.md:32]: Screen Shot 2021-03-23 at 6 13 52 PM
We should be able to edit the tier right from the equipment overview page. We don't need to change this from json editor.

**Comment by jaredmullane** [comment]
Tier selection available via NMS - deleting branch.
